api/config/index.ts

```diff
@@ -4,10 +4,12 @@ import { application } from './application.config'
 
 import * as aat from './environments/aat.config'
 import * as local from './environments/local.config'
+import * as preview from './environments/preview.config'
 
 export const configs = {
     aat,
     local,
+    preview,
```

Comment:
will comma on end of this throw exception? should prob be removed

Reply:
nope that's the standard for node es6/typecript linting
https://eslint.org/docs/rules/comma-dangle